src/frontend/src/icp/components/core/IcLoaderWallets.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+	import { onDestroy } from 'svelte';
+	import { initIcrcWalletWorker } from '$icp/services/worker.icrc-wallet.services';
+	import { initIcpWalletWorker } from '$icp/services/worker.icp-wallet.services';
+	import type { WalletWorker } from '$icp/types/ic-listener';
+	import { enabledIcNetworkTokens } from '$lib/derived/network-tokens.derived';
+	import type { TokenId } from '$lib/types/token';
+	import { debounce, nonNullish } from '@dfinity/utils';
+	import { toastsError } from '$lib/stores/toasts.store';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	let workers: Map<TokenId, WalletWorker> = new Map<TokenId, WalletWorker>();
+
+	const manageWorkers = async () => {
+		const unusedWorkers: TokenId[] = Array.from(workers.keys()).filter(
+			(tokenId) => !$enabledIcNetworkTokens.some(({ id: icTokenId }) => icTokenId === tokenId)
+		);
+
+		unusedWorkers.forEach((tokenId) => {
+			const worker = workers.get(tokenId);
+
+			if (nonNullish(worker)) {
+				worker.stop();
+				workers.delete(tokenId);
+			}
+		});
+
+		await Promise.all(
+			$enabledIcNetworkTokens.map(async (token) => {
+				if (!workers.has(token.id)) {
+					try {
+						const worker = await (token.standard === 'icrc'
+							? initIcrcWalletWorker(token)
+							: initIcpWalletWorker());
+
+						worker.stop();
+						worker.start();
+
+						workers.set(token.id, worker);
+					} catch (err: unknown) {
+						// We show the error to the user, but we keep loading the rest of the workers
+						toastsError({
```

Comment:
I'm a bit afraid that users ends with a wall of error toasts. Would it be more user friendly to already think at potential issue and display a single error toast if one or more worker did not load?

Reply:
Before this PR, they were not treated in the component actually, the user didn't have any feedback at component level.

The idea here was only to make the promises continue without fail, but using `allSettled` should be enough, so I can remove the error if you prefer.

Or do you prefer to show the error either way (in that case yes, I will do a single one)

Reply:
Before this PR, if I'm not wrong, no error was thrown if a worker was not loaded correctly therefore there wasn't a risk of "wall of toasts" or am I missing something?

Reply:
correct! so, if we want to keep the same behavior I can remove the toastError. Shall I remove it?

Reply:
romved